README.md

```diff
@@ -1,38 +1,47 @@
-* Uses the built in **commentstring** buffer option to define comment markers
-* Where a marker doesn't have a **space** character as padding this is added, configurable
-** This can be disabled in the options, see below but if useful when workig with numerous linters
-* Supports motions
-
-When the plugin is called it works out the range to comment/uncomment; if all lines in the given range are commented then it uncomments, otherwise it comments the range. This is useful when commenting a block out for testing with a real like comment in it; as for the plugin a comment is a comment.
-
-# Usage
+Toggle comments in Neovim, using built in **commentstring** filetype option;
+written in Lua. Without a doubt this plugin **is not required** and is a rip off
+of [TPope's Commentary](https://github.com/tpope/vim-commentary) with less
+features! What makes this plugin stand out over the numerous other comment
+plugins written in Lua are:
+
+- Comments each line, rather than adds block comments; making it easier to
+  toggle code when debugging
+- Uses the built in **commentstring** buffer option to define comment markers
+- Where a marker doesn't have a **space** character as padding this is added,
+  configurable \*\* This can be disabled in the options, see below but if useful
+  when working with numerous linters
+- Supports motions
+- Dot `.` repeatable
+
+When the plugin is called it works out the range to comment/uncomment; if all
+lines in the given range are commented then it uncomments, otherwise it comments
+the range. This is useful when commenting a block out for testing with a real
+like comment in it; as for the plugin a comment is a comment.
+
+## Usage
 
 Either use the command `CommentToggle`, e.g.:
 
-* `CommentToggle` comment/uncomment current line
-* `67,69CommentToggle` comment/uncomment a range
-* `'<,'>CommentToggle` comment/uncomment a visual selection
+- `CommentToggle` comment/uncomment current line
+- `67,69CommentToggle` comment/uncomment a range
+- `'<,'>CommentToggle` comment/uncomment a visual selection
 
 Or use the default mappings:
 
-* `gcc` comment/uncomment current line, this does not take a count, if you want a count use the `gc{count}{motion}`
-* `gc{motion}` comment/uncomment selection defined by a motion:
-** As lines are commented, any comment toggling actions will default to a linewise.
-** `gcc` comment/uncomment current line
-** `gcip` comment/uncomment a paragraph
-** `gc4w` comment/uncomment current line
-** `gc4j` comment/uncomment 4 lines below the current line
+- `gcc` comment/uncomment current line, this does not take a count, if you want
+  a count use the `gc{count}{motion}`
+- `gc{motion}` comment/uncomment selection defined by a motion: \*\*As lines are
+  commented, any comment toggling actions will default to a linewise.
+  - `gcip` comment/uncomment a paragraph
+  - `gc4w` comment/uncomment current line
+  - `gc4j` comment/uncomment 4 lines below the current line
 
-## Configure
+### Configure
```

Comment:
happy with current heading level,

Reply:
It was reported by linter as skipping heading levels.

Reply:
Only unwanted side effect here is that GitHub generates link in TOC, that might not be something you want as it gets populated. Perhaps a list then?

Reply:
not a bad point actually, alright fair enough